.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
:thinking:  why are you setting the server on vscode settings is this for any code or just have local 5500 runing with something else??